packages/fiori-elements-writer/src/data/templateAttributes.ts

```diff
@@ -1,5 +1,8 @@
 import { OdataVersion, TemplateType } from '../types';
 
+// https://ui5.sap.com/versionoverview.html
+export const oldestSupportedUI5Version = '1.38.0';
```

Comment:
@tobiasqueck This is lower than Fiori tools supports for FE V2 today. It might impact start-local, themes, ui5 libs used etc... in the templates.

Reply:
In `main` the value is still `1.0.0`, so I think the `1.38.0` is already an improvement. What higher version would you suggest and why?

Reply:
Should I use `1.71` as it is the lowest supported by the spec (https://www.npmjs.com/package/@sap/ux-specification)?

Reply:
@tobiasqueck We use `1.65.0` currently in tool-suite `ui5-info` as min ui5 FE V2. 
Since 1.71.0 is the supported version per https://ui5.sap.com/versionoverview.html and spec at this stage it would probably make sense to update tool-suite to 1.71.0. @IainSAP @theStiggySpd @ianquigley-sap - Do you agree?

Reply:
Changed it to `1.65.0` for now to stay consistent with the Fiori tools, however, I will provide another PR next week with a proposal to cleanup the versioning.